### DISCUSSION TITLE
Can anyone crack the maximum upload size limit for looping videos, it looks like it's encrypted or variable, about the 25 MiB limit.

### DISCUSSION THREAD

**Discussion by xiaoqy** [datarhei/restreamer#792] [Q&A]
![image](https://github.com/user-attachments/assets/fc416328-fa0d-498c-a9a6-2ea044102917)
![image](https://github.com/user-attachments/assets/68a3aad8-f76e-4e6c-b3b0-6c1d055b1bbd)
![image](https://github.com/user-attachments/assets/e8e5b5f8-41eb-4348-a8a3-8af87c2f0ef5)
Can anyone crack the maximum upload size limit for looping videos, it looks like it's encrypted or variable, about the 25 MiB limit.

**Comment by bluekiwidev** [comment]
This is preatty late but theck out this: #745
